src/core/config.js

```diff
@@ -58,7 +58,27 @@ const config = {
 	callbacks: {},
 
 	// The storage module to use for reordering tests
-	storage: localSessionStorage
+	storage: localSessionStorage,
+
+	// Get the testIds for all tests
+	// Simulates how test.js generates testIds by
+	// iteratively pushing testNames to get uniqueTestName
+	get testIds() {
```

Comment:
@trentmwillis I have a vague sense that using a getter here may bite us long-term in terms of confusion and flexibility. What's your take on this? Would making this a regular function be easier to understand/document/maintain long-term? Drawback is that it introduces a non-serialisable property to `QUnit.config`. On the other hand, if config does get serialised, the value it would currently serialise is likely wrong by the time it is read.

Maybe that's a reason to mount it somewhere else entirely, e.g. on `currentModule`.

Reply:
Just trying to get some traction here.

Reply:
@trentmwillis Any thoughts? I'd love to see this submitted to help tests run faster in parallel.

Reply:
Sorry, I've been busy working through other OSS tickets. From a brief glance, I think I'm inclined to agree that it doesn't belong on `QUnit.config`. I'll review more thoroughly this Friday.

Reply:
So after reviewing more thoroughly, I think this should just be a normal property that is an array.

The `testId` for each test is already generated eagerly (i.e., when you call `QUnit.test`). The information is already available within the (undocumented) `QUnit.config.modules` property but would require some munging. So instead, we could set this up such that as each testId gets initialized, we push it into a new `QUnit.config.testIds` property.